### PR TITLE
ensure python-configobj is present

### PR DESCRIPTION
**Pull request by richm**
tuned recently switched from configobj to configparser.  The role cannot
yet switch to use configparser.  Ensure that the package python-configobj
is present.

### CODE REVIEW

**Review by nhosoi — APPROVED** [review]
lgtm

File: vars/CentOS_7.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+__kernel_settings_packages: ["tuned", "python-configobj"]
+__kernel_settings_services: ["tuned"]
```
Comment (spetrosi) [inline vars/CentOS_7.yml:3]: Why do you create new vars files for CentOS_7 and RedHat_7?

Comment (richm) [reply vars/CentOS_7.yml:3]: The package is named `python3-configobj` on every platform except EL7.

**Review by spetrosi — APPROVED** [review]
lgtm

File: vars/CentOS_7.yml (hunk 1 above)
Comment (spetrosi) [reply vars/CentOS_7.yml:3]: Oh yes, I am blind

**Pull request merged**